### DISCUSSION TITLE
Let's start

### DISCUSSION THREAD

**Discussion by Mohmad-Naser-alnakeshbandi** [The-Escape-Box/TheEscapeBox#8] [General]
# Welcome to The Escape Box Blog
Greetings and welcome to The Escape Box Blog! Join us on an exciting journey as we delve into the development process of our latest creation, "The Escape Box." Here, we'll share insights into our progress, the creative thinking behind the game, and meticulously document our journey forward. We value your input and invite you to become an integral part of this game's development by leaving your feedback and comments!

# About the Game
Prepare to immerse yourself in an exhilarating 3D First Person gaming experience like no other. In "The Escape Box," you embark on a riveting quest from humble beginnings as a fledgling hero, striving to attain power and freedom from the confinements of a labyrinthine maze.

Your path to liberation is paved with challenges; solve intricate puzzles and engage in combat against formidable foes to enhance your abilities. Along the way, discover opportunities to upgrade your weaponry, turning once formidable enemies into mere obstacles on your hero's journey.

But make no mistake, progress comes at a price! Brace yourself to tackle an array of themed puzzles and confront a myriad of adversaries as you traverse through this thrilling adventure.

# Team Introduction
The team behind TheEscapeBox is small but with an vision. And we will do our best to share our fun by making this game with you all!

Lukas Weber and Mohamad Naser Alnakshbandi commence this project with equitable responsibilities. As project managers, gameplay designers, creative directors, sound-effect specialists, and much more, we jointly undertake diverse roles. As we deepen our understanding of Unity and the intricacies of game development, roles may evolve to encompass more specific responsibilities.

For our weekly sprints we are committed on spending 10h per person on developing an epic game experience

---

Time
Mohammad: 10h
Lukas: 10h
---

Please feel free to share your thoughts on the project in this and all subsequent posts. We will consistently update you on the steps we have taken and our plans for the future. Our aim is to fulfill both our own and your expectations.

**Comment by tonoizer** [comment]
Exciting journey ahead! Can't wait to see how 'The Escape Box' unfolds. Wishing the entire team all the best as they embark on this epic adventure. Looking forward to following along and offering feedback along the way!

Team: AetherArena

**Comment by TimWortmann** [comment]
Hey there, Escape Box team!
I just wanted to drop by and express my excitement for what you're creating with "The Escape Box." Reading about the dedication and passion you're pouring into this project truly resonates with me as someone who loves immersive gaming experiences. The concept of starting as a humble hero in a labyrinthine maze and striving for power and freedom is incredibly compelling. It's the kind of journey I've always craved in a game - one filled with challenges, puzzles, and the opportunity to grow and evolve along the way. I can't wait to follow along with your progress and be part of this journey. Keep up the amazing work, and know that you've got a whole community cheering you on every step of the way!

Team
Cooking Souls
